Crash at 90% build

## Build Process
Using yaourt -S polybar-git (same issues with yaourt -S polybar


**If you use some other way (like the AUR):**

## Build Log
==> Building and installing package
==> Making package: polybar-git 3.2.0-1 (Mon 15 Oct 2018 04:53:22 PM CEST)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Cloning polybar git repo...
Cloning into bare repository '/tmp/yaourt-tmp-magnasie/aur-polybar-git/polybar'...
remote: Enumerating objects: 300, done.
remote: Counting objects: 100% (300/300), done.
remote: Compressing objects: 100% (245/245), done.
remote: Total 16508 (delta 182), reused 109 (delta 55), pack-reused 16208
Receiving objects: 100% (16508/16508), 7.88 MiB | 1.34 MiB/s, done.
Resolving deltas: 100% (12302/12302), done.
==> Validating source files with md5sums...
    polybar ... Skipped
==> Extracting sources...
  -> Creating working copy of polybar git repo...
Cloning into 'polybar'...
done.
==> Starting prepare()...
Submodule 'lib/i3ipcpp' (https://github.com/jaagr/i3ipcpp) registered for path 'lib/i3ipcpp'
Submodule 'lib/xpp' (https://github.com/jaagr/xpp) registered for path 'lib/xpp'
Cloning into '/tmp/yaourt-tmp-magnasie/aur-polybar-git/src/polybar/lib/i3ipcpp'...
Cloning into '/tmp/yaourt-tmp-magnasie/aur-polybar-git/src/polybar/lib/xpp'...
Submodule path 'lib/i3ipcpp': checked out 'd4e4786be35b48d72dc7e59cf85ec34a90d129b5'
Submodule path 'lib/xpp': checked out '00165e1a6d5dd61bc153e1352b21ec07fc81245d'
==> Starting pkgver()...
==> Updated version: polybar-git 3.2.0.r50.gabdd5f92-1
==> Starting build()...
-- Trying to enable ccache
-- Couldn't locate ccache, disabling ccache...
-- The C compiler identification is Clang 7.0.0
-- The CXX compiler identification is Clang 7.0.0
-- Check for working C compiler: /usr/bin/clang
-- Check for working C compiler: /usr/bin/clang -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/clang++
-- Check for working CXX compiler: /usr/bin/clang++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- No build type specified; using RelWithDebInfo
-- Using supported compiler Clang-7.0.0
-- Found PkgConfig: /usr/bin/pkg-config (found version "1.5.3") 
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - not found
-- Check if compiler accepts -pthread
-- Check if compiler accepts -pthread - yes
-- Found Threads: TRUE  
-- Checking for module 'cairo-fc'
--   Found cairo-fc, version 1.15.14
-- Checking for module 'alsa'
--   Found alsa, version 1.1.6
-- Checking for module 'libcurl'
--   Found libcurl, version 7.61.1
-- Checking for module 'libnl-genl-3.0'
--   Found libnl-genl-3.0, version 3.4.0
-- Checking for module 'libpulse'
--   Found libpulse, version 12.2
-- Checking for module 'xcb-xkb'
--   Found xcb-xkb, version 1.13.1
-- Checking for module 'xcb-randr'
--   Found xcb-randr, version 1.13.1
-- Checking for module 'xcb-randr>=1.12'
--   Found xcb-randr, version 1.13.1
-- Checking for module 'xcb-xrm'
--   Found xcb-xrm, version 1.3
-- Checking for module 'xcb-cursor'
--   Found xcb-cursor, version 0.1.3
--  Build:
--    Type: RelWithDebInfo
--    CC: /usr/bin/clang -march=x86-64 -mtune=generic -O2 -pipe -fstack-protector-strong -fno-plt -O2 -g -DNDEBUG
--    CXX: /usr/bin/clang++ -march=x86-64 -mtune=generic -O2 -pipe -fstack-protector-strong -fno-plt -Wall -Wextra -Werror -O2 -pedantic -pedantic-errors -Wno-error=parentheses-equality -Wno-zero-length-array -O2 -g -DNDEBUG
--    LD: /usr/bin/ld -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now 
--  Targets:
-- [X]   polybar-msg
-- [ ]   testsuite
--  Module support:
-- [X]   alsa (1.1.6)
-- [X]   curl (7.61.1)
-- [X]   i3
-- [ ]   mpd
-- [X]   network (libnl) (3.4.0)
-- [X]   pulseaudio (12.2)
-- [X]   xkeyboard (1.13.1)
--  X extensions:
-- [X]   xcb-randr (1.13.1)
-- [X]   xcb-randr (monitor support) (1.13.1)
-- [ ]   xcb-render
-- [ ]   xcb-damage
-- [ ]   xcb-sync
-- [ ]   xcb-composite
-- [X]   xcb-xkb (1.13.1)
-- [X]   xcb-xrm (1.3)
-- [X]   xcb-cursor (0.1.3)
--  Log options:
-- [X]   Trace logging
-- Font not found: fixed:pixelsize=10
-- Font not found: unifont:fontformat=truetype
-- Font not found: siji:pixelsize=10
-- Checking for module 'xcb-proto'
--   Found xcb-proto, version 1.13
-- Found PythonInterp: /usr/bin/python2.7 (found suitable version "2.7.15", minimum required is "2.7") 
-- XCB[XCB]: Found component XCB
-- Found XCB_XCB: /usr/lib/libxcb.so  
-- XCB[ICCCM]: Found component ICCCM
-- Found XCB_ICCCM: /usr/lib/libxcb-icccm.so  
-- XCB[EWMH]: Found component EWMH
-- Found XCB_EWMH: /usr/lib/libxcb-ewmh.so  
-- XCB[UTIL]: Found component UTIL
-- Found XCB_UTIL: /usr/lib/libxcb-util.so  
-- XCB[IMAGE]: Found component IMAGE
-- Found XCB_IMAGE: /usr/lib/libxcb-image.so  
-- Found XCB: /usr/lib/libxcb.so;/usr/lib/libxcb-icccm.so;/usr/lib/libxcb-ewmh.so;/usr/lib/libxcb-util.so;/usr/lib/libxcb-image.so  
-- Searching for xcbgen with python2
-- Searching for xcbgen with python2.7
-- Searching for xcbgen with python3
-- Found xcbgen in /usr/lib/python3.7/site-packages
-- XCB[RANDR]: Found component RANDR
-- Found XCB_RANDR: /usr/lib/libxcb-randr.so  
-- Found XCB: /usr/lib/libxcb.so;/usr/lib/libxcb-icccm.so;/usr/lib/libxcb-ewmh.so;/usr/lib/libxcb-util.so;/usr/lib/libxcb-image.so;/usr/lib/libxcb-randr.so  
-- XCB[XKB]: Found component XKB
-- Found XCB_XKB: /usr/lib/libxcb-xkb.so  
-- Found XCB: /usr/lib/libxcb.so;/usr/lib/libxcb-icccm.so;/usr/lib/libxcb-ewmh.so;/usr/lib/libxcb-util.so;/usr/lib/libxcb-image.so;/usr/lib/libxcb-randr.so;/usr/lib/libxcb-xkb.so  
-- xpp: including xcb proto randr.xml
-- xpp: including xcb proto xkb.xml
-- xpp: including xcb proto xproto.xml
-- Checking for module 'jsoncpp>=1.7.7'
--   Found jsoncpp, version 1.8.4
-- Configuring done
-- Generating done
-- Build files have been written to: /tmp/yaourt-tmp-magnasie/aur-polybar-git/src/polybar/build
[  1%] Generating ../../../lib/xpp/include/xpp/proto/x.hpp
[  2%] Generating ../../../lib/xpp/include/xpp/proto/randr.hpp
list: size, ['red', 'green', 'blue']
list: size, ['red', 'green', 'blue']
[  3%] Generating ../../../lib/xpp/include/xpp/proto/xkb.hpp
Scanning dependencies of target xpp
[  4%] Linking CXX static library libxpp.a
[  4%] Built target xpp
Scanning dependencies of target i3ipc++
[  5%] Building CXX object lib/i3ipcpp/CMakeFiles/i3ipc++.dir/src/ipc.cpp.o
[  6%] Building CXX object lib/i3ipcpp/CMakeFiles/i3ipc++.dir/src/ipc-util.cpp.o
[  7%] Linking CXX static library libi3ipc++.a
[  7%] Built target i3ipc++
Scanning dependencies of target polybar-msg
[  8%] Building CXX object bin/CMakeFiles/polybar-msg.dir/ipc.cpp.o
[  9%] Building CXX object bin/CMakeFiles/polybar-msg.dir/utils/env.cpp.o
[ 10%] Building CXX object bin/CMakeFiles/polybar-msg.dir/utils/file.cpp.o
[ 12%] Building CXX object bin/CMakeFiles/polybar-msg.dir/utils/string.cpp.o
[ 13%] Linking CXX executable polybar-msg
[ 13%] Built target polybar-msg
Scanning dependencies of target polybar
[ 14%] Building CXX object bin/CMakeFiles/polybar.dir/adapters/alsa/control.cpp.o
[ 15%] Building CXX object bin/CMakeFiles/polybar.dir/adapters/alsa/mixer.cpp.o
[ 16%] Building CXX object bin/CMakeFiles/polybar.dir/adapters/net.cpp.o
[ 17%] Building CXX object bin/CMakeFiles/polybar.dir/adapters/net_nl.cpp.o
[ 18%] Building CXX object bin/CMakeFiles/polybar.dir/adapters/pulseaudio.cpp.o
[ 19%] Building CXX object bin/CMakeFiles/polybar.dir/cairo/utils.cpp.o
[ 20%] Building CXX object bin/CMakeFiles/polybar.dir/components/bar.cpp.o
[ 21%] Building CXX object bin/CMakeFiles/polybar.dir/components/builder.cpp.o
[ 23%] Building CXX object bin/CMakeFiles/polybar.dir/components/command_line.cpp.o
[ 24%] Building CXX object bin/CMakeFiles/polybar.dir/components/config.cpp.o
[ 25%] Building CXX object bin/CMakeFiles/polybar.dir/components/controller.cpp.o
[ 26%] Building CXX object bin/CMakeFiles/polybar.dir/components/ipc.cpp.o
[ 27%] Building CXX object bin/CMakeFiles/polybar.dir/components/logger.cpp.o
[ 28%] Building CXX object bin/CMakeFiles/polybar.dir/components/parser.cpp.o
[ 29%] Building CXX object bin/CMakeFiles/polybar.dir/components/renderer.cpp.o
[ 30%] Building CXX object bin/CMakeFiles/polybar.dir/components/screen.cpp.o
[ 31%] Building CXX object bin/CMakeFiles/polybar.dir/components/taskqueue.cpp.o
[ 32%] Building CXX object bin/CMakeFiles/polybar.dir/drawtypes/animation.cpp.o
[ 34%] Building CXX object bin/CMakeFiles/polybar.dir/drawtypes/iconset.cpp.o
[ 35%] Building CXX object bin/CMakeFiles/polybar.dir/drawtypes/label.cpp.o
[ 36%] Building CXX object bin/CMakeFiles/polybar.dir/drawtypes/progressbar.cpp.o
[ 37%] Building CXX object bin/CMakeFiles/polybar.dir/drawtypes/ramp.cpp.o
[ 38%] Building CXX object bin/CMakeFiles/polybar.dir/events/signal_emitter.cpp.o
[ 39%] Building CXX object bin/CMakeFiles/polybar.dir/events/signal_receiver.cpp.o
[ 40%] Building CXX object bin/CMakeFiles/polybar.dir/main.cpp.o
[ 41%] Building CXX object bin/CMakeFiles/polybar.dir/modules/alsa.cpp.o
[ 42%] Building CXX object bin/CMakeFiles/polybar.dir/modules/backlight.cpp.o
[ 43%] Building CXX object bin/CMakeFiles/polybar.dir/modules/battery.cpp.o
[ 45%] Building CXX object bin/CMakeFiles/polybar.dir/modules/bspwm.cpp.o
[ 46%] Building CXX object bin/CMakeFiles/polybar.dir/modules/counter.cpp.o
[ 47%] Building CXX object bin/CMakeFiles/polybar.dir/modules/cpu.cpp.o
[ 48%] Building CXX object bin/CMakeFiles/polybar.dir/modules/date.cpp.o
[ 49%] Building CXX object bin/CMakeFiles/polybar.dir/modules/fs.cpp.o
[ 50%] Building CXX object bin/CMakeFiles/polybar.dir/modules/github.cpp.o
[ 51%] Building CXX object bin/CMakeFiles/polybar.dir/modules/i3.cpp.o
[ 52%] Building CXX object bin/CMakeFiles/polybar.dir/modules/ipc.cpp.o
[ 53%] Building CXX object bin/CMakeFiles/polybar.dir/modules/memory.cpp.o
[ 54%] Building CXX object bin/CMakeFiles/polybar.dir/modules/menu.cpp.o
[ 56%] Building CXX object bin/CMakeFiles/polybar.dir/modules/meta/base.cpp.o
[ 57%] Building CXX object bin/CMakeFiles/polybar.dir/modules/network.cpp.o
[ 58%] Building CXX object bin/CMakeFiles/polybar.dir/modules/pulseaudio.cpp.o
[ 59%] Building CXX object bin/CMakeFiles/polybar.dir/modules/script.cpp.o
[ 60%] Building CXX object bin/CMakeFiles/polybar.dir/modules/systray.cpp.o
[ 61%] Building CXX object bin/CMakeFiles/polybar.dir/modules/temperature.cpp.o
[ 62%] Building CXX object bin/CMakeFiles/polybar.dir/modules/text.cpp.o
[ 63%] Building CXX object bin/CMakeFiles/polybar.dir/modules/xbacklight.cpp.o
[ 64%] Building CXX object bin/CMakeFiles/polybar.dir/modules/xkeyboard.cpp.o
[ 65%] Building CXX object bin/CMakeFiles/polybar.dir/modules/xwindow.cpp.o
[ 67%] Building CXX object bin/CMakeFiles/polybar.dir/modules/xworkspaces.cpp.o
[ 68%] Building CXX object bin/CMakeFiles/polybar.dir/utils/bspwm.cpp.o
[ 69%] Building CXX object bin/CMakeFiles/polybar.dir/utils/command.cpp.o
[ 70%] Building CXX object bin/CMakeFiles/polybar.dir/utils/concurrency.cpp.o
[ 71%] Building CXX object bin/CMakeFiles/polybar.dir/utils/env.cpp.o
[ 72%] Building CXX object bin/CMakeFiles/polybar.dir/utils/factory.cpp.o
[ 73%] Building CXX object bin/CMakeFiles/polybar.dir/utils/file.cpp.o
[ 74%] Building CXX object bin/CMakeFiles/polybar.dir/utils/http.cpp.o
[ 75%] Building CXX object bin/CMakeFiles/polybar.dir/utils/i3.cpp.o
[ 76%] Building CXX object bin/CMakeFiles/polybar.dir/utils/inotify.cpp.o
[ 78%] Building CXX object bin/CMakeFiles/polybar.dir/utils/io.cpp.o
[ 79%] Building CXX object bin/CMakeFiles/polybar.dir/utils/process.cpp.o
[ 80%] Building CXX object bin/CMakeFiles/polybar.dir/utils/socket.cpp.o
[ 81%] Building CXX object bin/CMakeFiles/polybar.dir/utils/string.cpp.o
[ 82%] Building CXX object bin/CMakeFiles/polybar.dir/utils/throttle.cpp.o
[ 83%] Building CXX object bin/CMakeFiles/polybar.dir/x11/atoms.cpp.o
[ 84%] Building CXX object bin/CMakeFiles/polybar.dir/x11/background_manager.cpp.o
[ 85%] Building CXX object bin/CMakeFiles/polybar.dir/x11/connection.cpp.o
[ 86%] Building CXX object bin/CMakeFiles/polybar.dir/x11/cursor.cpp.o
[ 87%] Building CXX object bin/CMakeFiles/polybar.dir/x11/ewmh.cpp.o
[ 89%] Building CXX object bin/CMakeFiles/polybar.dir/x11/extensions/randr.cpp.o
[ 90%] Building CXX object bin/CMakeFiles/polybar.dir/x11/extensions/xkb.cpp.o
/tmp/yaourt-tmp-magnasie/aur-polybar-git/src/polybar/src/x11/extensions/xkb.cpp:211:12: error: local variable 'name' will be copied despite being returned by name [-Werror,-Wreturn-std-move]
    return name;
           ^~~~
/tmp/yaourt-tmp-magnasie/aur-polybar-git/src/polybar/src/x11/extensions/xkb.cpp:211:12: note: call 'std::move' explicitly to avoid copying
    return name;
           ^~~~
           std::move(name)
1 error generated.
make[2]: *** [bin/CMakeFiles/polybar.dir/build.make:960: bin/CMakeFiles/polybar.dir/x11/extensions/xkb.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:576: bin/CMakeFiles/polybar.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
==> ERROR: A failure occurred in build().
    Aborting...
==> ERROR: Makepkg was unable to build polybar-git.


```

## Environment:
* Distro (with Version):
4.18.10-arch1-1-ARCH

--- a/src/x11/extensions/xkb.cpp
+++ b/src/x11/extensions/xkb.cpp
@@ -208,7 +208,7 @@ namespace xkb_util {
     if (string_util::contains(LAYOUT_SYMBOL_BLACKLIST, ";" + name + ";")) {
       return "";
     }
-    return name;
+    return move(name);
   }
 }
 

